packages/ui/src/components/shared/menu/ExportWalletDialog.tsx

```diff
@@ -30,11 +37,16 @@ export default function ExportWalletDialog(): JSX.Element {
   const { t } = useTranslation();
   const { open, doOpen, doClose } = useDialog();
   const { keyring } = useWalletState();
-  const [backup, setBackup] = useState<WalletQrBackup>();
+  const [backup, setBackup] = useState<WalletBackup>();
+  const [method, setMethod] = useState<ExportWalletMethod>(ExportWalletMethod.QRCode);
+
+  const switchMethod = (_: any, method: ExportWalletMethod) => {
+    setMethod(method);
```

Comment:
We should reset the method back to `QR Code` on closing the dialog similar to the `ExportAccountDialog`